app/assets/javascripts/dashboard.js

```diff
@@ -0,0 +1,27 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // Get all "navbar-burger" elements
+  const $navbarBurgers = Array.prototype.slice.call(document.querySelectorAll('.navbar-burger'), 0);
+  // Check if there are any navbar burgers
+  if ($navbarBurgers.length > 0) {
+    // Add a click event on each of them
+    $navbarBurgers.forEach( el => {
+      el.addEventListener('click', () => {
+        // Get the target from the "data-target" attribute
+        const target = el.dataset.target;
+        const $target = document.getElementById(target);
```

Comment:
data target is a selector, so use of `getElementById` would be incorrect for class/tag selectors
better to use `document.querySelector(target)`

Reply:
If I use querySelector I have got an error: "Uncaught TypeError: Cannot read property 'classList' of null  at HTMLAnchorElement.el.addEventListener"